### ISSUE TITLE
[Menu design][Accessibility] VoiceOver does not announce menu item role

### ISSUE THREAD

**Issue by natastro** [mozilla-mobile/firefox-ios#23249]
## Prerequisites

- Turn on VoiceOver
- Turn on treatment-a of the iOS Menu Redesign
- Found on Firefox Beta iOS app 133.0

## Steps to reproduce

1. Open Firefox Beta iOS app
2. Open the menu from the hamburger menu icon button at the bottom of the screen
3. As the action sheet appears on screen, swipe through the menu items and observe VoiceOver behavior

### Expected behavior / Actual behavior

As mentioned throughout the [Figma file](https://www.figma.com/design/RFz9fYtotQCQuinwcZujZt/%5B%5D-Menu-Redesign?node-id=19728-177454&t=ispcxT90kORu5xuz-4), there are menu items that should be announced by VoiceOver with the menu item role. They would require `role="menuitem"` (See [ARIA: menuitem role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menuitem_role) )

### Device & build information

* Device: iPhone 15, iOS 18.1
* Build version: Firefox Beta 133.0
* First seen version:  First time I've seen this issue
### Notes
Attachments: 

https://github.com/user-attachments/assets/b8ee7eb7-aab1-4aa4-8982-c92a1f706d14

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-10616)

**Comment by data-sync-user** [comment]
➤ Matt P commented:

Natalie StroudWinnie Teichmann  As far as I am aware, there is no “menuitem” option for UIAccessibilityTraits ( https://developer.apple.com/documentation/uikit/uiaccessibilitytraits ), and usually these clickable roles simply announce “Button.” Can we confirm if this is correct?

**Comment by data-sync-user** [comment]
➤ Matt P commented:

The only way to do this would be to add the “menu item” string as part of the full label

**Comment by data-sync-user** [comment]
➤ Natalie Stroud commented:

Hi Matt P - you are correct, my apologies. I had mixed Web and Mobile up by accident. I’ve gone ahead and canceled this ticket. Thank you!

**Comment by data-sync-user** [comment]
➤ Natalie Stroud commented:

Matt P - it is! I went ahead and canceled those both. Thank you!
